dev-docs/bidders/ads_interactive.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: bidder
+title: Ads Ineractive
+description: Prebid Ads Interactive Bidder Adapter
+biddercode: ads_interactive
+gpp_sids: usstate_all
+tcfeu_supported: false
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+deals_supported: false
+floors_supported: true
+fpd_supported: false
+ortb_blocking_supported: false
+media_types: banner, video, native
+multiformat_supported: will-bid-on-one
+userIds: all
+pbjs: true
+pbs: true
```

Comment:
prebid server adapters tend take longer before they are merged. If the JS adapter is merged earlier, you could change this to `false` and we could merge the docs earlier. A second PR with a change setting this to `true` would be required then :blush: